src/components/Info/_info.scss

```diff
@@ -0,0 +1,43 @@
+.info-section__heading {
+  @include carbon--type-style('productive-heading-03');
+}
+
```

Comment:
You're missing the info-card__heading class. See https://www.carbondesignsystem.com/tutorial/react/step-4/#type

Reply:
Its a typo "." is missing for .info-card__body in _info.scss
I

Reply:
I pushed the code to react-step-4 branch. Please verify and let me know.

Thanks

Reply:
You're still missing the info-card__heading class. See https://www.carbondesignsystem.com/tutorial/react/step-4/#type

Reply:
Added 
.info-section__heading {
  @include carbon--type-style('productive-heading-03');
}
In _info.scss

Hope this helps me.